pyquil/paulis.py

```diff
@@ -70,7 +70,7 @@ class PauliTerm(object):
     """A term is a product of Pauli operators operating on different qubits.
     """
 
-    def __init__(self, op: str, index: int, coefficient: Union[int, float, complex] = 1.0):
+    def __init__(self, op: str, index: int, coefficient: Union[int, float, complex] = 1.0) -> None:
```

Comment:
I think you can drop the `-> None` return type annotation here.

Reply:
OK, it was my habit from Zulip's annotation (e.g. https://github.com/zulip/zulip/blob/master/zerver/forms.py#L92). I suppose this convention varies across projects. Will remove the `-> None`.